site/components/policy/policy-box/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import React from 'react';
+
+import Statements from '../statements';
+import { H2 } from '../../text';
+
+import styles from './style.css';
+
+type StatementT = {
+  body: Function,
+};
+
+export type PolicyT = {
+  policyIndex: number,
+  heading: string,
+  body?: Function,
+  statements?: Array<StatementT>,
+};
+
+const padNumber = (n: number) => (n < 10 ? '0' + n : +n);
+
+const PolicyBox = ({ heading, body, statements = [], policyIndex }: PolicyT) => {
```

Comment:
Could use defaultProps here?

Reply:
What would be the benefit (feel this is more concise and the effect is the same)?

Reply:
Seems to be a pattern encouraging by React.  Also, there are performance implications when done in non functional components, so I guess it makes for doing things in a single way, and is arguably more readable? Ymmv, just a thought!